lib/utils/import.js

```diff
@@ -9,11 +9,59 @@ const {
 } = require('../utils/types');
 
 module.exports = {
+  gatherImports,
+  getImportFromMap,
   getSourceModuleNameForIdentifier,
   getSourceModuleName,
   getImportIdentifier,
 };
 
+const IMPORTS_TO_GATHER = [
+  ['ember'],
+  ['@ember/object', 'computed'],
+  ['@ember/object', 'get'],
+  ['@ember/object', 'getProperties'],
+  ['@ember/object', 'getWithDefault'],
+];
+
+/**
+ * Gathers the names that common imports are imported under.
+ * @param {map<string,string>} imports map of each imported path/function to the name its imported under
+ * @returns {function} a visitor for ImportDeclaration nodes
+ */
+function gatherImports(imports) {
+  return function (node) {
+    IMPORTS_TO_GATHER.forEach((importToGather) => {
+      const [path, namedImport] = importToGather;
+      const key = getImportMapKey(path, namedImport);
+      if (node.source.value === path) {
+        imports.set(key, imports.get(key) || getImportIdentifier(node, path, namedImport));
```

Comment:
I think we should keep track of all times something has been imported, rather than just the first one.